job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -189,12 +101,77 @@ pipelineJob("${folderPath}/cron-meta-nightly-pipeline") {
     }
 }
 
+String rhbaNightlyStage(String version, String branch, String configBranch) {
+    return """
+        stage('trigger RHBA nightly job ${branch}') {
+            steps {
+                build job: 'rhba.nightly/${branch}', propagate: false, wait: true, parameters: [
+                        [\$class: 'StringParameterValue', name: 'KIE_GROUP_DEPLOYMENT_REPO_URL', value: 'https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/service/local/repositories/scratch-release-rhba-${getNexusFromVersion(version)}/content-compressed'],
+                        [\$class: 'StringParameterValue', name: 'UMB_VERSION', value: '${getUMBFromVersion(version)}'],
+                        [\$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: "${version}"],
+                        [\$class: 'StringParameterValue', name: 'CONFIG_BRANCH', value: "${configBranch}"],
+                        [\$class: 'BooleanParameterValue', name: 'SKIP_TESTS', value: true]
+                ]
+            }
+        }
+    """
+}
+
+String kogitoNightlyStage(String kogitoVersion, String kogitoBranch, String optaplannerVersion, String rhbaVersion, String rhbaVersionPrefix, String configBranch) {
+    return """
+        stage('trigger KOGITO nightly job ${kogitoVersion}') {
+            steps {
+                build job: 'kogito.nightly/${kogitoBranch}', propagate: false, wait: true, parameters: [
+                        [\$class: 'StringParameterValue', name: 'RHBA_MAVEN_REPO_URL', value: 'http://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8081/nexus/content/repositories/rhba-${getNexusFromVersion(rhbaVersion)}-nightly-with-upstream'],
```

Comment:
Maybe we should use HTTPS?

```suggestion
                        [\$class: 'StringParameterValue', name: 'RHBA_MAVEN_REPO_URL', value: 'https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/content/repositories/rhba-${getNexusFromVersion(rhbaVersion)}-nightly-with-upstream'],
```

Reply:
Let's try, at least we will be ready for more recent Maven versions :)

Reply:
Done.